### DISCUSSION TITLE
How to filter "Low Stock" based on Category or Location in the main dashboard

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#5331] [Q&A]
I would like to filter my "Low Stock" on the basis of Category/Location.
My products are in 2 different locations and have multiple categories under them.
Using filter options will help me decide which location (in my case godown) is running low on stock and refill them.

**Comment by SchrodingersGat** [comment]
@rohitsingal1 this is not currently configurable on the main dashboard. The best aproach would be to navigate to the detail page for that location and view stock from there

**Comment by rohitsingal1** [comment]
Ok will use the detail page for information on low stock.
